README.md

```diff
@@ -64,4 +64,4 @@ This project is brought to you thanks to:
 
 List of organizations that help the AzerothCore project:
 
-[![JetBrains](https://user-images.githubusercontent.com/75517/51205146-7f225c80-1905-11e9-82e0-835627be170d.png)](https://www.jetbrains.com/?from=AzerothCore)
+[![JetBrains](https://user-images.githubusercontent.com/75517/51205146-7f225c80-1905-11e9-82e0-835627be170d.png)](https://www.jetbrains.com/?from=AzerothCore)
```

Comment:
?

Reply:
don't know whats up maybe git get insane ?

Reply:
invisible character from MVS I guess, i don't remember the name, "bom mark" or smthg like that

Reply:
MVS on linux?  NICE!